src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -319,6 +325,141 @@ def fuzz(self,
+
+    # This initialization routime assumes that the GCS bucket contains the standard Fuchsia SDK, as well as:
+    # * /qemu-for-fuchsia/*
+    # * /.ssh/*
+    resources_path = os.path.join(os.getcwd(), self.fuchsia_bucket_name)
+
+    shell.create_directory_if_needed(resources_path)
+
+    if 'FUCHSIA_RESOURCES_PATH' not in os.environ:
+      logs.log_error("Could not find path for remote Fuchsia resources bucket (FUCHSIA_RESOURCES_PATH)!")
+
+    subbed_fuchsia_gsutil_command = [
+        param.replace("{fuchsia_resources_path}",
+                      environment.get_value('FUCHSIA_RESOURCES_PATH', ''))
+        .replace("{local_resources_path}", resources_path)
+        for param in self.fuchsia_gsutil_command_template
+    ]
+    subprocess.call(subbed_fuchsia_gsutil_command)
+
+    # Save paths for necessary commands later.
+    qemu_path = os.path.join(resources_path, 'qemu-for-fuchsia', 'bin',
+                             'qemu-system-x86_64')
+    os.chmod(qemu_path, 0o550)
+    kernel_path = os.path.join(resources_path, 'target', 'x64',
+                               'qemu-kernel.bin')
+    os.chmod(kernel_path, 0o644)
+    authorized_keys_path = os.path.join(resources_path, '.ssh',
+                                        'authorized_keys')
+    pkey_path = os.path.join(resources_path, '.ssh', 'pkey')
+    os.chmod(pkey_path, 0o400)
+    sharefiles_path = os.path.join(resources_path, 'qemu-for-fuchsia', 'share',
+                                   'qemu')
+
+    # The FVM is minimally sized to begin with; extend it to make room for ephemeral packages etc.
+    drive_path = os.path.join(resources_path, "extended_fvm.blk")
+    raw_fvm_path = os.path.join(resources_path, "target", "x64", "fvm.blk")
+    copy_fvm = ["cp", raw_fvm_path, drive_path]
+    subprocess.call(copy_fvm)
+    os.chmod(drive_path, 0o644)
+    fvm_tool_path = os.path.join(resources_path, "tools", "fvm")
+    os.chmod(fvm_tool_path, 0o500)
+    subprocess.call([fvm_tool_path, drive_path, "extend", "--length", "1G"])
+
+    # Need to bake keys into ZBI so we can SSH into it.
+    zbi_tool = os.path.join(resources_path, "tools", "zbi")
+    os.chmod(zbi_tool, 0o500)
+    fuchsia_zbi = os.path.join(resources_path, "target", "x64", "fuchsia.zbi")
+    initrd_path = os.path.join(resources_path, "fuchsia-ssh.zbi")
+    make_fuchsia_ssh_zbi_command = [
+        zbi_tool, "-o", initrd_path, fuchsia_zbi, "-e",
+        "data/ssh/authorized_keys=" + authorized_keys_path
+    ]
+    subprocess.call(make_fuchsia_ssh_zbi_command)
+    os.chmod(initrd_path, 0o644)
+
+    # TODO: Add a mechanism for choosing portnum dynamically.
+    portnum = "56339"
+
+    self.subbed_qemu_base_command = [
+        param.replace("{qemu}", qemu_path).replace("{kernel}", kernel_path)
+        .replace("{drive}", drive_path).replace("{initrd}", initrd_path)
+        .replace("{sharefiles}", sharefiles_path).replace("{portnum}", portnum)
+        for param in self.fuchsia_qemu_command_template
```

Comment:
Similar comment as the one above, can you just build the command here? IMO that would be more straightforward than current version